.travis.yml

```diff
@@ -12,7 +12,8 @@ install:
 - npm install
 - bower install
 after_success:
-- sed -i -- 's/SF:ember-frost-sidebar\/\(.*\)/SF:addon\/\1.js/' coverage/lcov.info && rm -f coverage/lcov.info--
```

Comment:
Yes that script doesnt add everything so the idea was to run the script to get the secure keys and copy the rest from the frost-sort example I pasted the link to in my earlier comment